cmd/server/main.go

```diff
@@ -192,12 +195,34 @@ func main() {
 		}).Fatal("setting up RPC connection to zcashd")
 	}
 
+	// Get the sapling activation height from the RPC
+	saplingHeight, blockHeight, chainName, branchID, err := common.GetSaplingInfo(rpcClient)
+	if err != nil {
+		log.WithFields(logrus.Fields{
+			"error": err,
+		}).Warn("Unable to get sapling activation height")
+	}
+
+	log.Info("Got sapling height ", saplingHeight, " chain ", chainName, " branchID ", branchID)
+
+	// Initialize the cache
+	cache := common.NewBlockCache(opts.cacheSize)
+
+	stopChan := make(chan bool, 1)
+
+	// Start the block cache importer at latestblock - 100k(cache size)
```

Comment:
This comment is out of date, the default value is 40,000. I suggest just: `latestblock - cache size`.